### ISSUE TITLE
Dead links in readme

### ISSUE THREAD

**Issue by mrbrett** [wagnefrede/xDrip--Tasker-Tizen-Watchface-Integration#12]
I noticed a couple links in the readme weren't working (for me at least). 

This one for downloading the watch faces
`https://taskernet.com/shares/?user=AS35m8mlt5xx0ME%2BtNTXK6rFnQPs1RUb38m9tui3GLZpydqGl3C7iL1YYOsaI8lQbmAxMf%2B4&id=Project%3AxDrip%2Pro +atchm`

and this one for instructions on the import
`https://github.com/wagnefrede/xDrip-and-AAPS-Notifications-for-TiZENOS-WEAROS-Watchfaces-Tasker-AutoNotification-Watchmaker/blob/master/Watchfaces/README.md`

Also, it might be because of a missing step inside one of those two links but I can't get any tasker info loaded into my watchmaker watch faces. I downloaded two of your watch faces inside the watchmaker app, as well as created my own and referenced a few of the variables. The variables inside tasker are populated with the correct values and seem to update from xdrip just fine. I also see the `WM Send Variable` fields on the tasker tasks. So strange. I will keep poking around but if you have any insight, it would be greatly appreciated! 

I really appreciate all this work you have done. Watch faces look great! Can't wait to get them functioning for myself.

**Comment by mrbrett** [comment]
In regards to my issue with watch maker not pulling tasker variables, I am getting this error message "java.net.ConnectException: Failed to connect to localhost/127.0.0.1:17580" in autoTools

**Comment by FreDiabetics** [comment]
I'll check and update  all dead links this weekend.

Your autotools error says that autotools isnt able to grab the values from xDrip+ local brodcast. You have to activate the local broadcast  like I described it in 1.1 Apps & Settings

**Comment by mrbrett** [comment]
Awesome, thanks for taking a look.

 In regards to my local setup, it seems as though my tasker was getting and updating all my xdrip variables just fine, but it was sending all them to watchmaker live wallpaper. I think I am confused on how to set a watchmaker live wallpaper as my watch face. Therefore, I couldn't get any of your watchmaker watch faces that are on the watchmaker store to work, but I did add some new variable send fields on tasker that send directly to watchmaker and I made my own basic watch face with my sugar levels and trend, etc. That seems to work great! Once again, thank you so so much for putting in the work on this to figure it all out. Having this all on my watch makes a huge difference for me.
